articles/introduction-to-r/index.md

```diff
@@ -0,0 +1,315 @@
+# Introduction to R Programming
+
+The data science community has been growing over the past decade. The tools used in the industry have been powering this unprecedented growth. Hence, learning a programming language like R empowers us to contribute to the field. This article will clarify all the basic concepts of R and look at the various concepts. Going through the article should give you sufficient information to start coding in R. 
+
+### What is R?
+
+R is a software environment developed for statistical and graphical computing. Developed at the Bell Labs, it is a modified implementation of [S language](https://en.wikipedia.org/wiki/S_%28programming_language%29). It provides many statistical techniques such as [linear modeling](https://techvidvan.com/tutorials/r-generalized-linear-models/), [non-linear modeling](https://analyticsindiamag.com/hands-on-guide-for-non-linear-regression-models-in-r/), [statistical tests](http://r-statistics.co/Statistical-Tests-in-R.html), [classification, and clustering](https://www.geeksforgeeks.org/introduction-to-machine-learning-in-r/) algorithms, etc. One of the greatest strengths of the R language is the large community that uses it. Therefore, many optimized libraries are readily available to make the lives of early programmers simple. 
+
+### RStudio
+
+Before we begin, we need to ensure if the R interpreter is installed on our local machine. In this article, we will focus on programming concepts. You can use online editors like [Repl.it](https://repl.it/languages/rlang) for this tutorial. For further information on installation, refer to this [article](https://techvidvan.com/tutorials/install-r/).
+
+The RStudio interface has four components that we make use of most. The RStudio terminal is shown below.
+
+![rstuido](rstudio.jpg)
+
+The four components are as follows:
+
+1. **Editor**: The editor is used to create R scripts. R scripts are executed using the run button or the console. The run button is present at the top of the RStudio window. To execute a file using the console, we need to go through the following steps:
+   1. Set the current working directory to the directory, including the R script. Use the command `setwd(dir)` to set the working directory to the directory of your choice. Use `getwd()` to get the current working directory
+   2. We use the `source` command to execute the script file of our choice. `source(file_name)` executes the entire file and gives the output in the console section.
+   ![](R-Script.jpg)
+2. **Console and Terminal**: The console in R is similar to the IDLE in Python. We can execute any command here. For this tutorial, you may use the console.
+   ![](console.jpg)
+3. **Environment Variables**: It gives us a list of all the variables stored and the corresponding variables stored in them. These variables are created at the time of execution of the R scripts.
```

Comment:
Above, you begin sections with "The editor" and "The console," respectively. Maintain that parallel style here by saying the environment variables window lists all the variables...

Reply:
Bold environment variables window